setup.cfg

```diff
@@ -15,12 +15,11 @@ classifiers =
     License :: OSI Approved :: BSD License
     Programming Language :: Objective C
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.5
-    Programming Language :: Python :: 3.6
```

Comment:
Since 3.5 and 3.6 are still included in the CI and tox configuration, and AFAICT the code is still fully 3.5-compatible, we should keep those versions listed in the metadata too.